docs/csharp/distinguish-delegates-events.md

```diff
@@ -48,7 +48,7 @@ properly sort the elements. LINQ queries must be supplied with delegates
 in order to determine what elements to return. Both used a design built
 with delegates.
 
-Consider the `OnProgress` event handler. It reports progress on a task.
+Consider the `Progress` event handler. It reports progress on a task.
```

Comment:
Should this be "the `Progress` event"? I think this paragraph talks about the event itself, not about some specific event handler subscibed to it.